clients/admin-ui/src/features/system/SystemInformationForm.tsx

```diff
@@ -228,6 +265,7 @@ const SystemInformationForm = ({
                 isRequired
                 label="System name"
                 tooltip="Give the system a unique, and relevant name for reporting purposes. e.g. “Email Data Warehouse”"
+                disabled={lockedForGVL}
```

Comment:
I don't think this one is supposed to be locked! https://ethyca.atlassian.net/wiki/spaces/PM/pages/2742648890/Dictionary+forms